apps/frontend/core-dapp/src/stores/TokensStore.ts

```diff
@@ -29,31 +29,36 @@ export type Token = EthToken | Erc20Token
 // - we could even save those validated addresses into localStorage so the next time user come back
 // - their favourite token will show in the list by default
 export class TokensStore {
+  static NATIVE_ETH = {
+    type: 'native',
+    iconName: 'eth',
+    name: 'ETH',
+    shortName: 'ETH',
+  } as const
+  readonly COLLATERAL = {
+    type: 'erc20',
+    iconName: 'preeth',
+    name: 'prePO ETH Balance',
+    shortName: 'preETH',
+    erc20: this.root.collateralStore,
+  } as const
+
   tradeTokens: Token[]
   depositTokens: Token[]
   constructor(private root: RootStore) {
     this.tradeTokens = [
+      // TODO: enable after wrapAndDepositAndTrade integration
+      // TokensStore.NATIVE_ETH,
       {
         type: 'erc20',
-        iconName: 'cash',
-        name: 'Cash Balance',
-        shortName: 'USD',
+        iconName: 'preeth',
+        name: 'prePO ETH Balance',
```

Comment:
latest copy should be `prePO Balance` only
```suggestion
        name: 'prePO Balance',
```